tiledb/sm/filesystem/s3.h

```diff
@@ -571,6 +639,10 @@ class S3 {
     /** The overall status of the multipart request. */
     Status st;
 
+    /** Tracks all global order write intermediate chunks that make up a >5mb
```

Comment:
Fix doc string to have the following format:
`/**`
` * Comment line 1`
` * Comment line 2`
` */`